include/exadg/functions_and_boundary_conditions/interface_coupling.h

```diff
@@ -196,6 +203,19 @@ class InterfaceCoupling
         }
       }
     }
+
+    // To improve robustness, make sure that not all entries of marked_vertices are false.
+    // Otherwise, points will simply not be found by RemotePointEvaluation and results will
+    // probably be wrong.
+    bool all_false = true;
+    for(auto iter : marked_vertices)
+    {
+      if(iter == true)
+        all_false = false;
+    }
+
+    if(all_false)
+      marked_vertices = std::vector<bool>(dof_handler_src_.get_triangulation().n_vertices(), true);
```

Comment:
I am not sure if it is preferable, but I would write this as:
```suggestion
    if (std::all_off(marked_vertices.begin(), marked_vertices.end(), false))
      std::fill(marked_vertices.begin(), marked_vertices.end(), true);
```
I am not entirely sure if the `bool` vectors support these operations as they are somewhat special compared to other types like `int`, but I hope they do.

Reply:
I think one has to write is slightly differently using a lambda.

Please check again.